packages/mobile/src/components/audio/Audio.tsx

```diff
@@ -746,6 +758,13 @@ export const Audio = () => {
     handlePlaybackRateChange()
   }, [handlePlaybackRateChange, playbackRate])
 
+  useEffect(() => {
+    // Stop playback if we have unloaded a uid from the player
+    if (previousUid && !uid && !playing) {
+      handleStop()
```

Comment:
Since we're only using this here, why not just take `TrackPlayer.reset` as a dependency and call it directly?

Reply:
I did this and the other change below for consistency with where we handle all of these this way, perhaps we can roll this into convo about redoing this whole thing because it smells errrwhere

https://github.com/AudiusProject/audius-protocol/blob/55383d6cb704cf4c124eab9c01ad1c1c1ccffbed/packages/mobile/src/components/audio/Audio.tsx#L733